rest/permissions.py

```diff
@@ -114,3 +114,19 @@ def has_permission(self, request, view):
                 RepoPermission.manage_taxonomy[0]
                 in get_perms(request.user, repo)
             )
+
+
+class ManageRepoMembersPermission(BasePermission):
+    """Checks manage_repo_users permission"""
+
+    def has_permission(self, request, view):
+        if request.method in SAFE_METHODS:
+            return True
+        try:
+            repo = get_repo(view.kwargs['repo_slug'], request.user.id)
+        except NotFound:
+            raise Http404()  # pragma: no cover
```

Comment:
There are three `pragma: no cover`s in this PR. It looks like all three could have coverage.